.github/workflows/ci.yml

```diff
@@ -0,0 +1,69 @@
+# Primary CI checks for Rollbar-PHP-Laravel.
+#
+# Test with act:
+#   brew install act
+#   act -P ubuntu-latest=shivammathur/node:latest
+#
+# @see https://github.com/nektos/act/issues/329
+name: Rollbar-PHP-Laravel CI
+
+# Fire this action on pushes to main branch (master) as well as pull requests
+# (push: without any configuration assumes this). Also, run every day at 02:42
+# GMT -- this catches failures from dependencies that update indepdently.
+on:
+  push:
+  schedule:
+    - cron: '42 2 * * *'
+
+jobs:
+  # Check that this runs on PHP on all versions we claim to support, on both
+  # UNIX-like and Windows environments, and that use both the lowest possible
+  # compatible version as well as the most-recent stable version. This will
+  # fail-fast by default, so we include our edgiest versions (currently 7.4)
+  # first as they're most likely to fail.
+  # @seealso https://freek.dev/1546
+  # @seealso https://www.dereuromark.de/2019/01/04/test-composer-dependencies-with-prefer-lowest/
+  php-tests:
+    strategy:
+      matrix:
+        php: [7.3, 7.2]
```

Comment:
I'm keeping the scope narrow on this PR, specifically to emulate the original Travis behavior as closely as possible. Since the [original Travis definition][1] tested only on 7.3 and 7.2, I have retained that compatibility. Future releases will sort all this out, so that all documented versions will be under test. That's a bigger fish to fry, so I'm doing that in chunks.

[1]:https://github.com/rollbar/rollbar-php-laravel/pull/107/files#diff-6ac3f79fc25d95cd1e3d51da53a4b21b939437392578a35ae8cd6d5366ca5485L4-L7

Reply:
Sounds good.